workspaces/sonarqube/plugins/sonarqube/src/components/SonarQubeCard/RatingCard.tsx

```diff
@@ -57,14 +57,14 @@ export const RatingCard = ({
 }: {
   leftSlot: ReactNode;
   rightSlot: ReactNode;
-  title: string;
+  title?: string;
   titleIcon?: ReactNode;
   link: string;
 }) => {
   const classes = useStyles();
 
   return (
-    <Link to={link} color="inherit" underline="none">
+    <Link to={link} color="inherit" underline="none" target="_blank">
```

Comment:
The Link component will open in a new window/tab for any external URL automatically

Reply:
Thanks for pointing it.
Done:)